### ISSUE TITLE
How can I test output commands using helm and keeping the file ordination during execution?

### ISSUE THREAD

**Issue by lucasjct** [kyverno/chainsaw#1133]
### Describe your question

Hello everyone! Thanks for chainsaw, is a great tools! 🥳 

I'm attempting to test the output for several Helm commands. I'm conducting these tests separately for each file. However, my last file verifies the output for uninstalling the application. This has posed a challenge because the uninstallation should be the final test command, but it's being executed among other tests and disrupting the pipeline

How can I test it maintaining the files separated? 

Below is my current directories. The expected behavior, wold be run `9-uninstall-zora` like the last test?

```
├── 0-helm-install-test
│   └── chainsaw-test.yaml
├── 1-misc-test
│   └── chainsaw-test.yaml
 ....
├── 8-set-timout-for-scan
│   └── chainsaw-test.yaml
├── 9-uninstall-zora
│   └── chainsaw-test.yaml
├── config.yaml
```

My config file contains:
```
apiVersion: chainsaw.kyverno.io/v1alpha1
kind: Configuration
metadata:
  name: custom-config
spec:
  timeouts:
    apply: 45s
    assert: 20s
    cleanup: 45s
    delete: 25s
    error: 10s
    exec: 45s
  skipDelete: false
  failFast: true
  parallel: 1
```

and my test for helm uninstall:

```
apiVersion: chainsaw.kyverno.io/v1alpha1
kind: Test
metadata:
  name: delete-zora
spec:
  steps:
  - try:
    - script:
     #You can uninstall Zora and its components by uninstalling the Helm chart installed.
        content: helm uninstall zora -n zora-system
        check:
        # This check ensures that zora has been uninstalled correctly.
          ($stdout): |- 
            release "zora" uninstalled
```

Thank you very much!

### chainsaw version Version

v0.1.8

### Additional context

I'm using like reference this part of documentation:
https://kyverno.github.io/chainsaw/latest/examples/test-output/

**Comment by eddycharly** [comment]
@lucasjct tests are independent from each other, not sure how this is supposed to work 🤔 

can you elaborate on what you are doing ?

**Comment by lucasjct** [comment]
@eddycharly Thank you for the response.

What I've been doing is running tests to validate the output of helm commands. I start with the helm install and validate the program's output, then test some helm parameter passes and also validate the output. Finally, I run the helm uninstall and check if the resources were correctly removed.

However, the tests are executed independently, but they do not respect the ordering of the files I planned in the directory. So, instead of executing the file containing helm uninstall last, it ends up being executed in the middle of the tests, breaking the expected flow.

To work around this problem, we created a directory to execute only the uninstall script. And I run the test on GitHub actions by first calling all the tests and then calling a step lastly just for uninstalling.

My directories are structured as follows:

```
├── tests
│   ├── 0-helm-install-test
│   │   └── chainsaw-test.yaml
│   ├── 1-misc-test
│   │   └── chainsaw-test.yaml
│   ├── 2-vuln-test
│   │   └── chainsaw-test.yaml
│   ├── 3-change-schedule
│   │   └── chainsaw-test.yaml
│   ├── 4-custom-check
│   │   ├── chainsaw-test.yaml
│   │   └── check.yaml
│   ├── 5-computer-resources
│   │   └── chainsaw-test.yaml
│   ├── 6-retain-issues
│   │   └── chainsaw-test.yaml
│   ├── 7-large-vulnerability-reports
│   │   └── chainsaw-test.yaml
│   ├── 8-set-timout-for-scan
│   │   └── chainsaw-test.yaml
│   └── 9-expansive-scan
│       └── chainsaw-test.yaml
├── uninstall-zora
│   └── chainsaw-test.yaml
└── wait-scan.sh
```

**Comment by eddycharly** [comment]
@lucasjct this is expected behaviour, all tests are independent from each other.

From one i understand it should be a single test with different steps.

**Comment by lucasjct** [comment]
Hello, thank you for the response.

My intention, even though the tests are independent of each other, was for them to be executed following the numerical index within the files in the test directory (following different steps). However, when I run them, the order I structured in the directory is not being followed.

But I managed to create a second test directory, which is only used to terminate the test by executing the Helm Uninstall command. Therefore, it runs the chainsaw test twice in the pipeline, but we managed to work around the initial problem.

I will close this PR with the solution we used. Thank you for your attention! Much appreciated!

How we are executing:
```docker
 - name: Run tests
        run: |
          cd tests && \
          chainsaw test  --config ../config/config.yaml

      - name: Validate helm uninstall
        run: |
          cd uninstall-zora && \
          chainsaw test  --config ../config/config.yaml
     ```
